scripts/linters/other_files_linter.py

```diff
@@ -190,6 +190,15 @@ def check_third_party_libs_type_defs(
 
                 if lib_version[0] == '^':
                     lib_version = lib_version[1:]
+                # In cases where the version is in the form of git commit hashes
+                # such as 'git+https://github.com/username/repo#commit-hash',
+                # we extract the commit hash and use it as the version.
+                elif lib_version[:3] == 'git':
```

Comment:
Maybe check for `git+https://github.com/`

Reply:
should we use regex here?